_includes/navbar.html

```diff
@@ -1,31 +1,31 @@
-<nav class="navbar navbar-expand-lg navbar-light bg-light">
-    <a class="navbar-brand" href="#">Navbar</a>
+<nav class="navbar navbar-expand-lg navbar-light bg-dark">
+        <img class="resize__Img" src="{{"assets/img/logo-1.png" | relative_url}}" alt="logo">
     <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav"
         aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
         <span class="navbar-toggler-icon"></span>
     </button>
     <div class="collapse navbar-collapse" id="navbarNav">
-        <ul class="navbar-nav">
+        <ul class="m-0 navbar-nav">
             <li class="nav-item active">
-                <a class="nav-link" href="#">Services <span class="sr-only">(current)</span></a>
+                <a class="nav-link text-danger" href="#services">Services <span class="sr-only">(current)</span></a>
             </li>
             <li class="nav-item">
-                <a class="nav-link" href="#">Portfolio</a>
+                <a class="nav-link text-danger" href="#portfolio">Portfolio</a>
             </li>
             <li class="nav-item">
-                <a class="nav-link" href="#">About</a>
+                <a class="nav-link text-danger" href="#about">About</a>
             </li>
             <li class="nav-item">
-                <a class="nav-link" href="#">Team</a>
+                <a class="nav-link text-danger" href="#team">Team</a>
             </li>
             <li class="nav-item">
-                <a class="nav-link" href="#">Partners</a>
+                <a class="nav-link text-danger" href="#partners">Partners</a>
             </li>
             <li class="nav-item">
-                <a class="nav-link" href="#">Blog</a>
+                <a class="nav-link text-danger" href="#blog">Blog</a>
```

Comment:
not an anchor but a link to other page
```suggestion
                <a class="nav-link text-danger" href="blog.html">Blog</a>                
```
add the relative url